reflectable/lib/src/builder_implementation.dart

```diff
@@ -5582,10 +5584,66 @@ bool _isPrivateName(String name) {
   return name.startsWith('_') || name.contains('._');
 }
 
-EvaluationResult _evaluateConstant(
-    LibraryElement library, Expression expression) {
-  return ConstantEvaluator((library as LibraryElementImpl).source, library)
-      .evaluate(expression);
+DartObject? _evaluateConstant(LibraryElement library, Expression expression) {
+  // !!!TODO!!! Analyzing without errors until this point.
```

Comment:
Is this something that needs to be addressed before landing?

Otherwise we should probably file an issue and format this as
```suggestion
  // TODO(https://github.com/google/reflectable/issues/<issue number>) Analyzing without errors until this point.
```

Reply:
Yes, certainly, `!!!TODO!!!` should never be committed. Thanks for catching that!

Reply:
Changed the function to `async`, now using `_severe` like everywhere else. No more TODOs. ;-)